providers/implementations/keymgmt/slh_dsa_kmgmt.c

```diff
@@ -115,7 +138,7 @@ static int slh_dsa_import(void *keydata, int selection, const OSSL_PARAM params[
     int include_priv;
     struct slh_dsa_import_st p;
 
-    if (!ossl_prov_is_running()
```

Comment:
I don't think this and later changes to replace `ossl_prov_is_running()` are required.  It's not possible to call this and later functions without having first called `slh_dsa_new_key()` which includes the self check call.

The same applies to `slh_dsa_sig.c` where only `slh_dsa_newctx()` needs the self test call but the others don't.